contributed_definitions/nyaml/NXpeak_model.yaml

```diff
@@ -0,0 +1,140 @@
+category: base
+doc: |
+  Peak fitting of a one-dimensional spectrum.
+symbols:
+  doc: |
+    The symbols used in the schema to specify e.g. dimensions of arrays.
+  n_peaks: |
+    Number of peaks.
+type: group
+NXpeak_model(NXprocess):
+  region_REGION(NXfit_region):
+    doc: |
+      Subset of the abscisssa that is used for fitting.
+      It is envisioned that regions are labeled as regions_0, regions_1, and so on.
```

Comment:
How do you perform fits of multiple regions? As global fit of all of them simultaneously, with shared parameters, or independently? In the second case, I would define one instance of NXfit per region, as they are independent then, no?

Reply:
see below.